access-protected-api-obo/README.md

```diff
@@ -0,0 +1,151 @@
+<!-- Keeping yaml frontmatter commented out for now
+---
+# Metadata required by https://docs.microsoft.com/samples/browse/
+# Metadata properties: https://review.docs.microsoft.com/help/contribute/samples/process/onboarding?branch=main#add-metadata-to-readme
+languages:
+- java
+page_type: sample
+name: "Java API written in Spring Boot that both protects its own endpoints and calls Graph on-behalf-of the client."
+description: "This Java API that protects it's own endpoints using JWT scope validation and calls Microsoft Graph on behalf of the client. The code in this sample is used by one or more articles on docs.microsoft.com."

```

Comment:
it's should be its

Reply:
Thanks, addressed in https://github.com/Azure-Samples/ms-identity-docs-code-java/pull/5/commits/1c104010698da35ee19a6b4ad528de2ba26df54b

Reply:
Marking resolved (@skabou doesn't have permission to do so)